crypto/BRCryptoAccount.c

```diff
@@ -186,23 +209,35 @@ if (bytesPtr > bytesEnd) return NULL; /* overkill */ \
     BRKey ethPublicKey;
     BRKeySetPubKey(&ethPublicKey, bytesPtr, 65);
     BYTES_PTR_INCR_AND_CHECK (65);
+    BREthereumAccount eth = createAccountWithPublicKey(ethPublicKey);
+
+    // XRP
+    size_t xrpSize = UInt32GetBE(bytesPtr);
+    BYTES_PTR_INCR_AND_CHECK (szSize);
 
-    return cryptoAccountCreateInternal (mpk, ethPublicKey, timestamp);
+    BRGenericAccount xrp = gwmAccountCreateWithSerialization (genericRippleHandlers->type, bytesPtr, xrpSize);
+    assert (NULL != xrp);
+
+    return cryptoAccountCreateInternal (mpk, eth, xrp, timestamp);
 #undef BYTES_PTR_INCR_AND_CHECK
 }
 
 static void
 cryptoAccountRelease (BRCryptoAccount account) {
-    accountFree(account->eth);  // Core holds???
 //    printf ("Account: Release\n");
+    accountFree(account->eth);
+    gwmAccountRelease(account->xrp);
+
```

Comment:
Not to block PR, but going forward (maybe tracked by an issue for us to do a pass), what do you think about zeroing (or 0xFF, whatever) out on release? Would help us catch use after free situations.

Reply:
CORE-415